nss-day-cohort-51/holiday-road-grand-canyon-gliders
# database update

# Description

I rewrote the database structure as a relational database with a normalized(flat) database.  Where before we were storing the biz/eat id's in the tripObj as an array, now the arrays have been removed from the tripObj, and instead a resource of joins connects each trip by id to any number of bizs/eats.  

## Type of change

Please delete options that are not relevant.


# Testing Instructions

We should test to make sure that this doesn't break any thing we had in place.  Be sure to delete your existing db.json before testing as this give an error.  



# Checklist:
